test/functional/test_framework/script.py

```diff
@@ -711,41 +709,42 @@ def sign_input_segwitv0(tx, input_index, input_scriptpubkey, input_amount, privk
 # Note that this corresponds to sigversion == 1 in EvalScript, which is used
 # for version 0 witnesses.
 def SegwitV0SignatureMsg(script, txTo, inIdx, hashtype, amount):
+    ZERO_HASH = bytes([0]*32)
```

Comment:
In commit "test: avoid unneeded hash -> uint256 -> hash roundtrips" (346a099fc1e282c8b53d740d65999d8866d17953)

Could simplify this to `ZERO_HASH = bytes(32)`